Backpack/lint-baseline.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<issues format="5" by="lint 4.2.2" client="gradle" variant="debug" version="4.2.2">
+
+    <issue
+        id="CustomViewStyleable"
+        message="By convention, the custom view (`MonthView`) and the declare-styleable (`BpkCalendar`) should have the same name (various editor features rely on this convention)"
+        errorLine1="    val a = this.context.obtainStyledAttributes(attrs, R.styleable.BpkCalendar, R.attr.bpkCalendarStyle, 0)"
+        errorLine2="                                                       ~~~~~~~~~~~~~~~~~~~~~~~">
+        <location
+            file="src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt"
+            line="117"
+            column="56"/>
+    </issue>
+
+    <issue
+        id="CustomViewStyleable"
+        message="By convention, the custom view (`MonthView`) and the declare-styleable (`BpkCalendar`) should have the same name (various editor features rely on this convention)"
+        errorLine1="    val a = this.context.obtainStyledAttributes(attrs, R.styleable.BpkCalendar, R.attr.bpkCalendarStyle, 0)"
+        errorLine2="                                                       ~~~~~~~~~~~~~~~~~~~~~~~">
+        <location
+            file="src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt"
+            line="117"
+            column="56"/>
+    </issue>
+
+    <issue
+        id="CustomViewStyleable"
```

Comment:
Is this the same issue as above?

Reply:
I can't see any difference between the three either, but if I remove the duplicates the lint errors come back 🤷‍♀️